docusaurus/docs/reactnative/basics/troubleshooting.mdx

```diff
@@ -340,3 +340,9 @@ private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
 };
 ...
 ```
+
+## Project won't build on a MacBook with Apple M1
+
+On newer MacBooks with the Apple M1 SoC, there are a few required steps that need to be followed for an app to build. The user [aiba](https://github.com/aiba) has written [a guide](https://github.com/aiba/react-native-m1/blob/33f71b8226b943fc8e236a08960584709cb74089/README.md) to make the necessary changes to your project.
```

Comment:
hey! what do you think about linking to the default branch instead of the permalink? 

i dont really like permalinks cause if you update things, users accessing through the permalink wont get the updated results.

the problem with regular links is that if something is moved to another folder, it will break.

i would rather have users complaining about a broken link that its easier to fix than users complaining about a tutorial that is not working and would require some more work on our side to understand that the problem is cause things are outdated...

what do you think of linking to `https://github.com/aiba/react-native-m1` instead?

Reply:
I'm okay with doing that of course, my thinking was that it'd avoid displaying the wrong information if things were to change in the repo, but I don't have a strong opinion on that.

I'll make the change. :)